### ISSUE TITLE
screen readers - tables and geomap

### ISSUE THREAD

**Issue by sarumim** [wet-boew/wet-boew#2625]
On Geomap page
http://wet-boew.github.io/wet-boew/demos/geomap/geomap-eng.html

1. I am wondering if the checkbox needs more information provided for screen readers as well as the Zoom to element in the 2 tables

when tabbing in both NVDA and JAWS I head
"checkbox not checked" > tab > "Toronto" > "Zoom to element" 

in NVDA the first time focus is on the first checkbox NDVA will read the whole table and add the additional information

2. the first table the text on the button is "Zoom to element" and the second table only has the icon but the title tag is used and is the same for both tables

**Comment by hsrudnicki** [comment]
@dfarough @LaurentGoderre @pjackson28 : can be this issue closed ?

**Comment by hsrudnicki** [comment]
Please update this issue's status? [Note that this issue had been created in 2013, and recently commented in 2014]

**Comment by hsrudnicki** [comment]
Due to a lack of further discussion or status update(s), this issue has been labeled "Going Stale" and closed.
